healthz.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	maxIdleConnection = 10
+	maxTimeoutSec     = 4
+)
+
+type HealtzConfig struct {
+	ApiEndpoint    string
+	ApiCertPath    string
+	ApiCACertPath  string
+	ApiKeyPath     string
+	EtcdEndpoint   string
+	EtcdCertPath   string
+	EtcdCACertPath string
+	EtcdKeyPath    string
+	Port           int
+}
```

Comment:
I'd recommend comments on the config parameters, to explain what sort of string is expected here, including examples.

Reply:
Actually, the flags explain it quite well. Never mind.